vllm/spec_decode/spec_decode_worker.py

```diff
@@ -92,13 +92,13 @@ def __init__(
         # Lazy initiazliation.
         self.scorer: SpeculativeScorer
 
-    def init_device(self) -> None:
+    def init_device(self, spec_type: Optional[str] = None) -> None:
```

Comment:
can we remove this arg and pass it via config / init? we want to keep `init_device` consistent with other implementations of WorkeBase interface

Reply:
Done as suggested.